launch/master_slaves_pwm_rgb8.launch

```diff
@@ -0,0 +1,167 @@
+<!-- In this example launch file one camera creates a pwm signal on GPIO 2 which can be used to
```

Comment:
Did you test this example? I am pretty sure the `<?xml version="1.0"?>` needs to be the first line, so probably move that up.